src/Components/DataBatchContext.tsx

```diff
@@ -23,23 +27,27 @@ export const DataBatchContextProvider = connect(
     const configuredLimit = dataScope.limit() ?? 20
     const [limit, setLimit] = useState(configuredLimit)
     const [initialLimit, setInitialLimit] = useState(configuredLimit)
+    const [search, setSearch] = useState<string | undefined>()
```

Comment:
It looks like, that the `search` value sometimes is reused after navigating away from a page.

How to reproduce:
* Visit https://edit-beta.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://dave-data-search.scrivito-portal-app.pages.dev/en/equipment?_scrivito_workspace_id=s33bdf5568004455&_scrivito_display_mode=view
* Search for "clean"
* => only "Air cleaner" is shown
* Click on Orders

Expected: Open orders are visible.

Actual: No orders found.